configure.ac

```diff
@@ -173,19 +173,19 @@ AC_MSG_CHECKING([the symbolic links in the source tree])
 AS_IF([test -L build/pkgs/liblzma/package-version.txt], [dnl OK
     AC_MSG_RESULT([intact])
```

Comment:
Is `build/pkgs/liblzma/package-version.txt`? Isn't it `build/pkgs/sagelib/src`?

Reply:
Perhaps it doesn't matter. But as we also expect that `build/pkgs/sagelib/src` is a symbolic link and because subsequent checks are also about `build/pkgs/sagelib/src`, it is a bit confusing...

Reply:
The tests are designed so that they also work when the `configure` script runs out of `pkgs/sage-conf_pypi`. In this distribution, there is no symlink `build/pkgs/sagelib/src`.

Reply:
OK. At least I can see that the test passes for ordinary sdist case at line 174 and `sage-conf_pypi` case at line 178, and otherwise something is wrong.